examples/seismic/tutorials/04_dask.ipynb

```diff
@@ -178,41 +180,40 @@
     "        'observed' data using the real model. For a real case\n",
     "        the worker would be reading seismic data from disk.\n",
     "    \"\"\"\n",
-    "    true_model = get_true_model()\n",
-    "    shot_id = param['shot_id']\n",
-    "    \n",
-    "    src_coordinates = np.empty((1, len(param['shape'])))\n",
-    "    src_coordinates[0, :] = [30, param['shot_id']*1000./(param['nshots']-1)]\n",
+    "    # Reconstruct objects\n",
+    "    with open(\"arguments.pkl\", \"rb\") as cp_file:\n",
+    "        cp = pickle.load(cp_file)\n",
+    "        \n",
+    "    op = cp['operator']\n",
+    "    geometry = cp['geometry']\n",
+    "    # source position changes according to the index\n",
+    "    shot_id=param['shot_id']\n",
     "    \n",
-    "    # Number of receiver locations per shot.\n",
-    "    nreceivers = 101\n",
-    "\n",
-    "    # Set up receiver data and geometry.\n",
-    "    rec_coordinates = np.empty((nreceivers, len(param['shape'])))\n",
-    "    rec_coordinates[:, 1] = np.linspace(0, true_model.domain_size[0], num=nreceivers)\n",
-    "    rec_coordinates[:, 0] = 980. # 20m from the right end\n",
+    "    geometry.src_positions[0,:]=[30, shot_id*1000./(param['nshots']-1)]\n",
+    "    true_d, _, _ = op.forward()\n",
+    "    dump_shot_data(shot_id, true_d, geometry)\n",
     "\n",
+    "def generate_shotdata(src_coor,rec_coor):\n",
+    "    true_model = get_true_model()\n",
     "    # Geometry \n",
-    "    geometry = AcquisitionGeometry(true_model, rec_coordinates, src_coordinates,\n",
+    "    geometry = AcquisitionGeometry(true_model, rec_coor, src_coor,\n",
```

Comment:
for homogeneity with the other notebooks, I'd keep the original variable names

Reply:
OK, Do I have to do something else?

Reply:
Probably yes. I have asked reviews from 4 of us, but nobody replied yet.

Reply:
> for homogeneity with the other notebooks, I'd keep the original variable names

I agree with this